### ISSUE TITLE
Migrate from `workflow-cps-global-lib` to `pipeline-groovy-lib`

### ISSUE THREAD

**Issue by daniel-beck** [jenkins-infra/helpdesk#3011]
### Service(s)

cert.ci.jenkins.io, ci.jenkins.io, infra.ci.jenkins.io, release.ci.jenkins.io, trusted.ci.jenkins.io, weekly.ci.jenkins.io

### Summary

The useful feature we want has been extracted from `workflow-cps-global-lib` into `pipeline-groovy-lib`. Time to uninstall the former, and whatever plugins are now unnecessary (probably `git-server`).

**Comment by dduportal** [comment]
- https://github.com/jenkins-infra/kubernetes-management/pull/2543 => applying to weekly.ci and infra.ci
- https://github.com/jenkins-infra/kubernetes-management/pull/2542 => applying to release.ci

**Comment by dduportal** [comment]
Great job @lemeurherve : the warning message disappeared from infra.ci and release.ci.

Let's do the other next week!

**Comment by dduportal** [comment]
Removed from trusted.ci by @lemeurherve and @smerle33 🎉

**Comment by daniel-beck** [comment]
> Removed from trusted.ci by @lemeurherve and @smerle33 🎉

`git-server` is still present, no for no discernible reason.

**Comment by lemeurherve** [comment]
> `git-server` is still present, now for no discernible reason.

You're right, I didn't thought about checking for it in trusted, now removed too.
